cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -920,8 +920,8 @@ New Cypher command for administering privilege for listing constraints.
 | `CYPHER runtime=compiled` (Compiled runtime) | Functionality | Removed| Replaced by the new `pipelined` runtime which covers a much wider range of queries.
 | `CREATE INDEX [name] FOR (n:Label) ON (n.prop)` | Syntax | Added | New syntax for creating indexes, which can include a name.
 | `CREATE CONSTRAINT [name] ON ...` | Syntax | Extended | The create constraint syntax can now include a name.
-| `DROP INDEX name` | Syntax | Added | New command for dropping an index by name.
-| `DROP CONSTRAINT name` | Syntax | Added | New command for dropping a constraint by name, no matter the type.
+| `DROP INDEX name` | Syntax | Added | <<query-plan-drop-index-by-name,New command>> for dropping an index by name.
+| `DROP CONSTRAINT name` | Syntax | Added | <<query-plan-drop-constraint-by-name,New command>> for dropping a constraint by name, no matter the type.
```

Comment:
These links just go to the query plans for the commands, I would have added links to the syntax for DROP in the index/constraint chapter instead. Don't know what the exact links should be now with the restructuring of that part of the Cypher manual, but here in the published version https://neo4j.com/docs/cypher-manual/4.3/administration/constraints/#administration-constraints-drop-constraint and corresponding for indexes

Reply:
Thanks, @Lojjs I will address these links. Do you know who would be best to reach out to in the team to discuss the best place to link these in 4.4 onwards?

Reply:
I think that would be @davidoliverSP2 or maybe @martin-neotech  in your team since they have been involved in the restructuring